conference/user_panel.py

```diff
@@ -496,7 +496,13 @@ def get_tickets_for_current_conference(user):
     return Ticket.objects.filter(
         Q(fare__conference=conference.code)
         & Q(orderitem__order___complete=True)
-        & Q(user=user)
+        & (Q(user=user) | Q(orderitem__order__user__pk=user.assopy_user.pk))
+    ).annotate(
+        is_buyer=Case(
+            When(orderitem__order__user__pk=user.assopy_user.pk, then=Value(True)),
+            default=Value(False),
+            output_field=BooleanField(),
+        )
```

Comment:
This is definitely nicer, but in this case we were probably fine with N+1 because we're looking at only one users tickets. However please don't change it back, it's better this way. :)

Reply:
The main concern here was to keep the business logic outside of the templates; instead the `if ticket.is_buyer` is a much cleaner check, it's obvious what we're checking for. Performance is a side benefit :)

Reply:
True, but keeping out of templates could be solved for example by method on a model.